worlds/tunic/__init__.py

```diff
@@ -406,8 +404,8 @@ def pre_fill(self) -> None:
             if len(viable_locations) < self.amount_to_local_fill:
                 raise OptionError(f"TUNIC: Not enough locations for local_fill option for {self.player_name}. "
                                   f"This is likely due to excess plando or priority locations.")
-
-            self.fill_locations += viable_locations
+            self.fill_locations = []
+            self.fill_locations += viable_locations[:self.amount_to_local_fill]
```

Comment:
```suggestion
            self.fill_locations = viable_locations[:self.amount_to_local_fill]
```

Should be the same result.

Reply:
![image](https://github.com/user-attachments/assets/e9e90d7d-5bab-438f-83ef-17bb535e7ec3)
if only

Reply:
Well actually I guess it doesn't need to be a list[TunicLocation]